src/test/java/com/artipie/ArtipieServer.java

```diff
@@ -156,6 +156,17 @@ public ArtipieServer(final Path root, final String name, final String config,
      *  for usage in all these places.
      */
     public int start() throws IOException {
+        return this.start("flat");
+    }
+
+    /**
+     * Starts the server.
+     *
+     * @param layout Layout
+     * @return Port the servers listening on.
+     * @throws IOException In case of error creating configs or running the server.
+     */
+    public int start(final String layout) throws IOException {
```

Comment:
@olenagerasimova to me it does not seem logical that a configuration parameter is passed to `start` method. I'd suggest to specify layyou (or whole `artipie.yaml` content) when `ArtipieServer` is created. It might be too much to add in this PR, so I'd suggest to add a puzzle for that if you agree about this design issue.